js/libs/quantimodo-api.js

```diff
@@ -37,7 +37,7 @@ Quantimodo = function () {
                 dataType: 'json',
                 contentType: 'application/json',
                 beforeSend: function (xhr) {
-                    xhr.setRequestHeader("Authorization", "Bearer " + access_token);
+                    xhr.setRequestHeader('Authorization', 'Bearer ' + access_token);
```

Comment:
Line is too long.<br>Identifier 'access_token' is not in camel case.<br>'access_token' is not defined.